src/utils/shared_io_service.cpp

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "shared_io_service.h"
+
+#include <dsn/utility/flags.h>
+
+namespace dsn {
+namespace tools {
+
+DSN_DEFINE_uint32("core", timer_service_worker_count, 3, "the number of threads for timer service");
+DSN_DEFINE_validator(timer_service_worker_count, [](uint32_t worker_count) -> bool {
+    if (worker_count < 3) {
```

Comment:
add a const variable for '3'. Avoid to use immediate var

Reply:
> 

OK, I'll define a const variable to represent the minimum `timer_service_worker_count.